libsolidity/parsing/Parser.cpp

```diff
@@ -2094,7 +2099,7 @@ optional<string> Parser::findLicenseString(std::vector<ASTPointer<ASTNode>> cons
 	else
 		parserError(
 			3716_error,
-			{-1, -1, m_scanner->charStream()},
+			{-1, -1, m_scanner->currentLocation().sourceName},
```

Comment:
These also have the "entire file" special cases.

Reply:
Do we want to do something about them? From https://github.com/ethereum/solidity/pull/11596#discussion_r661681364 it looks like they're ok now?

Reply:
@chriseth What about this one? Are all these cases with `-1` fine now?